vscode-simple-contrib/package.json

```diff
@@ -37,7 +36,7 @@
 		"SAPOSS.app-studio-toolkit"
 	],
 	"dependencies": {
-		"@sap-devx/app-studio-toolkit-types": "1.0.0",
-		"@sap-devx/guided-development-types": "0.0.14"
+		"@sap-devx/app-studio-toolkit-types": "1.2.0",
+		"@sap-devx/guided-development-types": "0.0.13"
```

Comment:
whether is some incompatible changes in 0.0.14 ?

Reply:
0.0.14 is not yet published, the latest one is 0.0.13
it is a bug to use 0.0.14